schemas/Identity.json

```diff
@@ -0,0 +1,22 @@
+{
+    "type": "object",
+    "description": "The user properties to apply when identifying",
+    "properties": {
+      "eventName": {
+        "enum": ["Identity"]
```

Comment:
this is probably kind of unnecessary .. or should be $identify?

Reply:
it's only added because `eventName` is mandatory for the type gen, is completely unused by the clients, what's `$identity`?

Reply:
ah nm, I was thinking posthog's identify call lets you set user properties as in mixpanel, but its entirely separate